use-cases/vehicle-routing/src/main/java/org/acme/vehiclerouting/rest/VehicleRouteDemoResource.java

```diff
@@ -0,0 +1,225 @@
+import org.eclipse.microprofile.openapi.annotations.parameters.Parameter;
+import org.eclipse.microprofile.openapi.annotations.responses.APIResponse;
+import org.eclipse.microprofile.openapi.annotations.responses.APIResponses;
+import org.eclipse.microprofile.openapi.annotations.tags.Tag;
+
+@Tag(name = "Demo data", description = "Timefold-provided demo vehicle routing data.")
+@Path("demo-data")
+public class VehicleRouteDemoResource {
+
+    private static final String[] FIRST_NAMES = { "Amy", "Beth", "Chad", "Dan", "Elsa", "Flo", "Gus", "Hugo", "Ivy", "Jay" };
+    private static final String[] LAST_NAMES = { "Cole", "Fox", "Green", "Jones", "King", "Li", "Poe", "Rye", "Smith", "Watt" };
+    private static final int[] SERVICE_DURATION_MINUTES = { 10, 20, 30, 40 };
+    private static final LocalTime MORNING_WINDOW_START = LocalTime.of(8, 0);
+    private static final LocalTime MORNING_WINDOW_END = LocalTime.of(12, 0);
+    private static final LocalTime AFTERNOON_WINDOW_START = LocalTime.of(13, 0);
+    private static final LocalTime AFTERNOON_WINDOW_END = LocalTime.of(18, 0);
+
+    public enum DemoData {
+        PHILADELPHIA(0, 60, 6, 2, LocalTime.of(7, 30),
+                1,2,15, 30,
+                new Location(39.7656099067391, -76.83782328143754),
+                new Location(40.77636644354855, -74.9300739430771)),
+        HARTFORT(1,50, 6, 2, LocalTime.of(7, 30),
+                1,3,20, 30,
+                new Location(41.48366520850297, -73.15901689943055),
+                new Location(41.99512052869307, -72.25114548877427)),
+        FIRENZE(2,77, 6, 2, LocalTime.of(7, 30),
+                1, 2, 20, 40,
+                new Location(43.751466, 11.177210), new Location(43.809291, 11.290195));
+
+        private long seed;
+        private int customerCount;
+        private int vehicleCount;
+        private int depotCount;
+        private LocalTime vehicleStartTime;
+        private int minDemand;
+        private int maxDemand;
+        private int minVehicleCapacity;
+        private int maxVehicleCapacity;
+        private Location southWestCorner;
+        private Location northEastCorner;
+
+        DemoData(long seed, int customerCount, int vehicleCount, int depotCount, LocalTime vehicleStartTime,
+                 int minDemand, int maxDemand, int minVehicleCapacity, int maxVehicleCapacity,
+                 Location southWestCorner, Location northEastCorner) {
+            if (minDemand < 1) {
+                throw new IllegalStateException("minDemand (" + minDemand + ") must be greater than zero.");
+            }
+            if (maxDemand < 1) {
+                throw new IllegalStateException("maxDemand (" + maxDemand + ") must be greater than zero.");
+            }
+            if (minDemand >= maxDemand) {
+                throw new IllegalStateException("maxDemand (" + maxDemand + ") must be greater than minDemand ("
+                        + minDemand + ").");
```

Comment:
For readability, we've lately been moving towards the `"maxDemand (%s) ...".formatted(maxDemand)` pattern.

Reply:
Done.